### DISCUSSION TITLE
Deno compatibility

### DISCUSSION THREAD

**Discussion by kreha1** [dolanmiu/docx#1307] [Ideas / Requests]
Hi, could you make this library work with deno and publish it on https://deno.land/x ? If I can help in some way, please do tell what I can do.

**Comment by dolanmiu** [comment]
Yeah if you can work on it that would be great

How would it work on deno? I have zero clue

Edit: Figured out how to do it, had to add a web-hook

Done!

https://deno.land/x/docx

**Reply by dolanmiu** [reply]
I get this error though:

https://dev.to/pandaquests/how-do-i-set-the-version-for-a-deno-module-450p

But they say it resolves itself after a day, so lets wait

**Reply by dolanmiu** [reply]
It is now on deno, just checked
